cms/search/wagtail_hooks.py

```diff
@@ -0,0 +1,41 @@
+from typing import TYPE_CHECKING
+
+from wagtail import hooks
+from wagtail.models import Page
```

Comment:
nitpick: move to `if TYPE_CHECKING`

Reply:
Damn it, imported it for that purpose, ended up not moving it! Done!